portfolio/src/main/java/com/google/sps/servlets/ListCommentServlet.java

```diff
@@ -20,28 +20,35 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import com.google.sps.data.Comment;
+import java.lang.*;
```

Comment:
prefer to import the specific library you need, as opposed to *.

Reply:
Alright! Thank you for the tip.